src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -809,7 +809,7 @@ void goto_analyzer_parse_optionst::help()
 {
   std::cout <<
     "\n"
-    "* * GOTO-ANALYZER " CBMC_VERSION " - Copyright (C) 2017 ";
```

Comment:
I don't think the output will be nicely aligned anymore. You will need to change the lines below as well.

Reply:
ok, I updated the lines below ending on `* *` , too